packages/server-core/src/hooks/convert-video.ts

```diff
@@ -1,363 +0,0 @@
-export {}
```

Comment:
we should keep this as a reference, as we are wanting to bring it back at some point